README.md

```diff
@@ -1 +1,202 @@
+
+
+
+- **[Home_Page_is_Designed]**
+- **[About_Section_is_Designed]**
+- **[Feature_Section_is_Designed_using_javascript]**
+- **[Responsive_version_is_Made]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<!-- Note: Right now Application hasn't been Deployed yet. In future if this will be deployed,
+Deployed Link will update here -->
+- [Live Demo Link]  <a href="#"><button type="button" color:red background: white>Visit Site</button></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+A new developer could make use of this project by cloning the repository and could make adjustments
+in the code accordingly.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+1- Install Visual Studio Code<br>
+2- Install Live server Extension in Visual studio code
+ 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:/zunairkhan811/Web-development-course-Project.git
+```
+
+
+### Install
+
+Install this project with:
+
+
+--> Visual Studio Code
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+--> Click on live Link or Click on live server in Vs Studio.
+
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+In order to run for linter errors, you will have to create a pull request
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+
+--> By purchasing domain and hosting
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Author**
+
+- GitHub: [@zunairkhan811](https://github.com/zunairkhan811)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Will_try_update_html_file]**
+- [ ] **[Will_try_to_update_css_file]**
+- [ ] **[will_try_to_update_javascript_file]**
+- [ ] **[will_try_to_add_contact_section]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project kindly follow us and give rating by clicking on star button to this repository.
+Thanks!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse and Cindy Shin in Behance design for inspiring me enough to make this project
+
```

Comment:
- [ ] Please make sure that you give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) under this `Acknowledgments` section :pray: